### PR TITLE
[Doc] fix examples

### PR DESCRIPTION
**Pull request by Rhett-Ying**
## Checklist
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] Code is well-documented
- [ ] To the best of my knowledge, examples are either not affected by this change,
      or have been fixed to be compatible with this change
- [ ] Related issue is referred in this PR
- [ ] If the PR is for a new model/paper, I've updated the example index [here](../examples/README.md).

### CODE REVIEW

**Review by VoVAllen — APPROVED** [review]
Thanks

File: python/dgl/nn/pytorch/conv/appnpconv.py [hunk 1]
```diff
@@ -45,17 +45,17 @@ class APPNPConv(nn.Module):
...
-            1.0000],
-            [1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000,
-            1.0000],
-            [1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000,
-            1.0000],
-            [1.0303, 1.0303, 1.0303, 1.0303, 1.0303, 1.0303, 1.0303, 1.0303, 1.0303,
-            1.0303],
-            [0.8643, 0.8643, 0.8643, 0.8643, 0.8643, 0.8643, 0.8643, 0.8643, 0.8643,
-            0.8643],
+    >>> print(res)
```
Comment (mufeili) [inline python/dgl/nn/pytorch/conv/appnpconv.py:48]: @Rhett-Ying By the way, dropout is non-deterministic and it's natural to get different results from multiple calls.

Comment (Rhett-Ying) [reply python/dgl/nn/pytorch/conv/appnpconv.py:48]: oh, I see. thanks...
